jcl/src/java.base/share/classes/java/lang/invoke/MethodHandle.java

```diff
@@ -963,6 +972,19 @@ private static final MethodHandle resolveInvokeDynamic(Class<?> clazz, String na
 		return result;
 	}
 	
+	/*
+	 * Retrieve the class name of the constant pool class element located at the specified
+	 * index in clazz's constant pool. Then, throw a NoClassDefFoundError with the cause
+	 * set as ClassNotFoundException. The message of NoClassDefFoundError and 
+	 * ClassNotFoundException contains the name of the class, which couldn't be found.
+	 */
+	private static Throwable noClassDefFoundError(Class<?> clazz, int index) {
```

Comment:
Its totally a nitpick, but can you call this `throwNoClassDefFoundError(...)`?   so it will match the naming convention in the j.l.i. package for these kinds of methods

Reply:
fixed